.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
 
 /.data
-/dist
\ No newline at end of file
+
+dist/
+src/types
```

Comment:
this type folder is not necessary?

Reply:
Yeah that folder is generated by  schema graphql when execute codegen command.